config/confighttp/README.md

```diff
@@ -34,6 +35,7 @@ exporter:
     headers:
       test1: "value1"
       "test 2": "value 2"
+    compression: gzip
```

Comment:
If we are making a recommendation, the value from this example should follow the recommendation.

Reply:
done